feature-libs/product-configurator/rulebased/core/facade/configurator-groups.service.ts

```diff
@@ -269,6 +269,28 @@ export class ConfiguratorGroupsService {
     return this.getNeighboringGroupId(owner, 1);
   }
 
+  /**
+   * Returns the description of the next group.
+   *
+   * @param {Configurator.Configuration} configuration - Configuration
+   * @returns {string} the description of the next group
+   */
+  getNextGroupDescription(configuration: Configurator.Configuration): string {
```

Comment:
Also: It feels more natural (and is also more robust) to return an observable here. Because you are implicitly assuming that this.getNextGroupId is emitting instantly.
So we would deal with observables in all layers up to the UI and only there use async pipe to subscribe

Reply:
done.